index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <div id="navbar">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+</div>
+  <section id="welcome-section">
+<h1>welcome</h1>
+</section>
+<section id="projects">
+  <p1 class="project-tile">this is mohammed and roland</p1>
+  <a>this is our project</a>
+</section>
+<a id="profile-link" target="_blank"
```

Comment:
This <a> tag would definitely work better with a href attribute applied to it. The value of this attribute will be the link that the user will be directed to when he clicks on this link. Learn more about anchor tags [here](https://html.com/anchors-links/#:~:text=The%20hypertext%20reference%2C%20or%20href,.com.)

Reply:
- [ ] Please kindly be more explicit, let the person know in clear term that they suppose to add `href` attribute to the anchor tag. 👍